src/R/Components/Impl/InteractiveWorkflow/Implementation/RInteractiveEvaluator.cs

```diff
@@ -207,5 +218,13 @@ public sealed class RInteractiveEvaluator : IInteractiveEvaluator {
                 _coreShell.DispatchOnUIThread(() => CurrentWindow.WriteLine(message));
             }
         }
+
+        private void VisualElement_SizeChanged(object sender, System.Windows.SizeChangedEventArgs e) {
+            int width = (int)(e.NewSize.Width / CurrentWindow.TextView.FormattedLineSource.ColumnWidth);
+            // From R docs:  Valid values are 10...10000 with default normally 80.
+            _terminalWidth = Math.Max(10, Math.Min(10000, width));
+
+            Session.OptionsSetWidth(_terminalWidth);
```

Comment:
Can we rename the method to `...Async` too, so it's more obvious that it needs to be awaited or otherwise observed?